addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -155,3 +155,89 @@ def test_mailing_list_merge_cornercase(self):
         merge = merge_form.save()
         self.assertEqual(merge.src_list_ids, self.mailing_list_1 + self.mailing_list_2)
         self.assertEqual(merge.dest_list_id, self.mailing_list_3)
+
+
+class TestMailingContactImport(MassMailCommon):
+    """Test the transient <mailing.contact.import>."""
+
+    @users('user_marketing')
+    def test_mailing_contact_import(self):
+        first_list, second_list, third_list = self.env['mailing.list'].create([
+            {'name': 'First mailing list'},
+            {'name': 'Second mailing list'},
+            {'name': 'Third mailing list'},
+        ])
+
+        self.env['mailing.contact'].create([
+            {
+                'name': 'Already Exists',
+                'email': 'already_exists@example.com',
```

Comment:
already_exists_list_1 and already_exists_list_2 ? that way it is explicit when reading :)